### PR TITLE
Luscious Locks: Make new line separation evident

### PR DESCRIPTION
**Pull request by ayushr2**
Following up from https://piazza.com/class/jvd552xfs6q7ki?cid=49

~Please merge only after 29th Jan 2020 (we do not update dates while the assignment is up).~

### CODE REVIEW

File: lockpicking_lab.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ You should begin by running the lockpick and then starting the safe with `r <net
 
 ## Answer Submission
 
-There is a file in your repository called `submission.txt`. You should provide the answers to each phase of the safe in this file. Provide one answer per line. For example if you think the safe has three phases and the answers to the phases are `answer1`, `answer2`, `answer3` then your `submission.txt` should look like:
+There is a file in your repository called `submission.txt`. You should provide the answers to each phase of the safe in this file. Answers should be newline separated. For example if you think the safe has three phases and the answers to the phases are `answer1`, `answer2`, `answer3` then your `submission.txt` should look like:
```
Comment (PradyumnaShome) [inline lockpicking_lab.md]:
```suggestion
There is a file in your repository called `submission.txt`. You should provide the answers to each phase of the safe in this file. Answers should be separated by Unix newlines (`\n` also known as line feed (LF)). For example if you think the safe has three phases and the answers to the phases are `answer1`, `answer2`, `answer3` then your `submission.txt` should look like:
```

Comment (ayushr2) [reply lockpicking_lab.md]: We usually just use "newline-separated" other docs as well. Usually, it does not have double meaning?

Comment (PradyumnaShome) [reply lockpicking_lab.md]: I think it's better to be explicit, since for Windows, a newline means \r\n.
See https://en.wikipedia.org/wiki/Newline

**Comment by PradyumnaShome** [comment]
I am proposing a merge (after the requested changes are made) as soon as possible (instead of after the deadline expires), because in previous semesters there have been people who got the answer, but due to confusion, did not get points. 
I don't these people deserve to get penalized for this somewhat subtle detail that is not related to the learning objective of the assignment, and is more to do with how we grade things.

**Pull request merged**